tomviz/vtkNonOrthoImagePlaneWidget.cxx

```diff
@@ -749,8 +756,21 @@ void vtkNonOrthoImagePlaneWidget::OnMouseMove()
 {
   // See whether we're active
   //
-  if (this->State == vtkNonOrthoImagePlaneWidget::Outside ||
-      this->State == vtkNonOrthoImagePlaneWidget::Start) {
+  if (this->State == vtkNonOrthoImagePlaneWidget::Outside) {
+    return;
+  }
+
+  if (this->State == vtkNonOrthoImagePlaneWidget::Start) {
+    if (this->VoxelTimerId == -1) {
+      this->Interactor->AddObserver(vtkCommand::TimerEvent,
+                                    this->VoxelTimerCommand);
+      this->VoxelTimerId = this->Interactor->CreateOneShotTimer(100);
+    }
+    // Uncomment else block if event should only emitted after the mouse is done
+    // moving
+    // else {
+    //   this->Interactor->ResetTimer(this->VoxelTimerId);
+    // }
```

Comment:
Do we need this block?

Reply:
Since the logic is not necessarily easy to follow, I figure I might leave those instructions there.
Essentially, at the moment we are throttling the firing of the event, if we uncomment those lines we would be debouncing it.